src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image"
+import styles from "./Banner.module.scss"
+import BannerImage from "@/public/images/banner.png"
+import { Button } from "../shared/Button"
+
+export default function Banner() {
+    return (
+        <div className={styles.container}>
+            <Image className={styles.image} src={BannerImage} alt=""></Image>
+            <div className={styles.info}>
+                <div className={styles.all_text}>
+                    <p className={styles.text1}>Бытовая химия, косметика и хозтовары</p>
+                    <p className={styles.text2}>оптом по кокчетаву и области</p>
+                    <Button text="В КАТАЛОГ" icon_src="" icon_alt="" width={300} height={95}></Button>

```

Comment:
```suggestion
                    <Button
                      text="В КАТАЛОГ"
                      iconSrc=""
                      iconAlt=""
                      width={300}
                      height={95}
                    />

```
Если атрибутов больше двух, то переносить их на новую строчку. 
Атрибуты принято пистать таким образом iconSrc.
Не нужно передавать пустую строку. Если нужно, то можно задать пропсам в компоненте значения по умолчанию.